tests/mocha/lists_test.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+goog.module('Blockly.test.lists');
+
+const {runSerializationTestSuite} = goog.require(
+  'Blockly.test.helpers.serialization'
+);
```

Comment:
[I don't think you need to line wrap imports](https://google.github.io/styleguide/jsguide.html#es-module-imports). Can you try unwrapping and see if eslint is still happy?